test/transforms.jl

```diff
@@ -275,7 +279,8 @@
       p₂ = scatter(n.x, n.y, label="ZScore")
       p = plot(p₁, p₂, layout=(1,2))
 
-      @test_reference joinpath(datadir,"zscore.png") p
+      @test_reference joinpath(datadir, "zscore.png") p
+      # savefig(p, joinpath(datadir, "zscore.png"))
```

Comment:
```suggestion
```

Reply:
I removed the comments of savefig function.